lib/img/aws.js

```diff
@@ -45,6 +45,29 @@ AwsDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function awsDelete(key) {
```

Comment:
Echoing @pdehaan comments earlier, I don't think it'd hurt to sanity-check the key in the backends here, e.g. with a quick regex test to ensure it's alphanumeric.  In theory it's already validated at higher levels, but...